strapi/strapi
# feat: send diagnostic messages from remote strapi

### What does it do?

- listen to diagnostic messages from the server, and report them so we can log them on the cli and to a file
when using `--verbose`

### Why is it needed?

- So we can receive information about whats happening on the remote strapi during a transfer

### How to test it?

try this transfer, you should see logs from the `push-handler` and `local-destination-provider`

`yarn strapi transfer --to {STRAPI_APP_ADMIN_URL} --to-token {TRANSFER_TOKEN}  --verbose`
